### ISSUE TITLE
Custom Theme and AMPforWP Options

### ISSUE THREAD

**Issue by IvanPr** [MohammedKaludi/ampforwp-custom-theme#6]
When 'AMP Starter Theme' plugin is activated the AMPforWP Options should have the 'Design' tab inactive, since it has no effect on  'AMP Starter Theme' , or does it?

**Comment by MohammedKaludi** [comment]
@IvanPr Yes, I agree, for now, it does not have any controls, but in very near future I'm planning to add more features and controls in the theme. The idea is other developers can build AMP theme with this starter theme in future.
